### DISCUSSION TITLE
How categories will be different from the collections?

### DISCUSSION THREAD

**Discussion by matvance** [medusajs/medusa#3865] [Q&A]
Hello. I am thriving to start my journey with your awesome engine. However, I found it harder to organize my products than in any other engine because of the lack of relational categories. So, I have noticed that there is already a new beta functoin called product categories but I am not sure how is it different from collections which are already in the engine despite the possibility of being hierarchical. Or is there any other way to organize products into categories? 
This information would help me a lot since I am planning to open a store with thousands of products which won't be possible without products hierarchy like in any other store.

**Comment by riqwan** [comment]
Hey @komarnisky,

Based on what you've described here, Product Categories feature is what you're looking for - its hierarchical and you can use the products api to query based on categories. 

We have documentation on the differences here - https://docs.medusajs.com/modules/products/categories#product-category-vs-product-collection

Article on introducing product categories - https://medusajs.com/blog/announcing-product-categories/

Take a look and let me know if that answers your question.

**Comment by matvance** [comment]
Thank you, @riqwan! Is it ready to use or It is just a super-beta feature?
Is there any chance that this feature will be removed or change dramatically before the stable release?
